win/CS/HandBrakeWPF/HandBrakeWPF.csproj

```diff
@@ -41,8 +41,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Caliburn.Micro" Version="4.0.136-rc" />
-    <PackageReference Include="gong-wpf-dragdrop" Version="2.3.2" />
+    <PackageReference Include="Caliburn.Micro" Version="4.0.173" />

```

Comment:
Can you revert this upgrade?  

I was holding back off upgrading Caliburn due to package issues. 

No harm in leaving the drag/drop upgrade in place

Reply:
Sure I'll do that.